P5/Source/Specs/charName.xml

```diff
@@ -28,6 +29,15 @@ $Id$
     <textNode/>
     <!--s:assert> some schematron rules here</s:assert-->
   </content>
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
delete this entire `<constraintSpec>`. The deprecation Schematron check will (or at least should) be automatically generated at build time because of the `@validUntil` on the `<elementSpec>`.